packages/coinstac-ui/app/config.js

```diff
@@ -33,12 +33,15 @@ const conf = convict({
     noURLPrefix: true,
   },
   logFile: 'coinstac-log.json',
+  // these are appended to the home dir for you OS 
+  // *nix: ~/.config/coinstac
+  // win: C:\Users\username\AppData\Local\Temp\coinstac
   logLocations: {
-    darwin: '~/Library/Logs/coinstac/',
-    freebsd: '~/.config/coinstac/',
-    linux: '~/.config/coinstac/',
-    sunos: '~/.config/coinstac/',
-    win32: '$HOME/AppData/Roaming/coinstac/',
+    darwin: 'Library/Logs/coinstac/',
+    freebsd: '.config/coinstac/',
+    linux: '.config/coinstac/',
+    sunos: '.config/coinstac/',
+    win32: 'coinstac/',
```

Comment:
would be cool if something like [this](https://www.npmjs.com/package/appdirectory) were more robust.  there may be.  i didn't search very hard.